documentation/release_notes.rst

```diff
@@ -8,6 +8,54 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.3
+=============
+
+New Features
+------------
+-  Added the Range-based versions of the following algorithms: ``all_of``, ``any_of``, ``count``, ``count_if``, ``equal``, ``move``, ``remove``, ``remove_if``, ``replace``, ``replace_if``.
+-  Added the following utility ranges (viewes): ``generate``, ``fill``, ``rotate``.
+
+Changes to Existing Features
+-----------------------------
+-  Improved performance of the ``discard_block_engine`` (including ``ranlux24``, ``ranlux48``, ``ranlux24_vec``, ``ranlux48_vec`` predefined engines) and ``normal_distribution`` RNG functionality.
+- Added two constructors to ``transform_iterator``: ``transform_iterator()`` and ``transform_iterator(iter)``.
+- ``transform_iterator`` meets the requirements of forward iterators instead of random-access iterator requirements.
+
+Fixed Issues
+------------
+-  Fixed parallel execution of ``swap_ranges`` algorithm with following C++ standard execution policies: ``unseq``, ``par``.
+-  Fixed the issue of scan-based algorithms, which caused memory corruption and double memory free operation on CPU devices with -O0 and -g options.
```

Comment:
"Fixed an issue causing memory corruption and double freeing memory in scan-based algorithms on CPU devices"

Referring to specific options seems redundant to me, but OK to keep if others prefer so.

Reply:
This fix relates to debug mode, so it was decided to specify debug options to keep consistency with previous release notes.

Reply:
OK, then use this: "Fixed an issue causing memory corruption and double freeing in scan-based algorithms compiled with -O0 and -g options and run on CPU devices".